ui/nuxeo-pagination-controls.js

```diff
@@ -42,18 +44,55 @@ import { html } from '@polymer/polymer/lib/utils/html-tag.js';
             display: block;
           }
 
+          .container {
+            @apply --layout-horizontal;
+            @apply --layout-center;
+          }
+
           .controls {
             color: var(--nuxeo-text-default, #000);
             font-size: 1rem;
-            padding: 0 4px;
+            @apply --layout-horizontal;
+            @apply --layout-center;
           }
 
-          paper-icon-button {
-            padding: 8px;
+          nuxeo-select {
+            max-width: 5rem;
+
+            --paper-dropdown-menu-input: {
+              padding: 0;
+              min-width: 2rem;
+              text-align: center;
+            }
+
+            --paper-input-container: {
+              padding: 0;
+            }
+
+            --paper-input-container-underline: {
+              display: none;
+            }
+
+            --paper-input-container-underline-focus: {
+              display: none;
+            }
+
+            --paper-input-container-shared-input-style: {
+              font-size: 1rem;
+              vertical-align: bottom;
+              margin: 1px 0 0 0;
+              width: inherit;
+              max-width: 5rem;
+            }
+          }
+
+          .total {
+            margin-inline-start: 2rem;
+            font-size: 1rem;
           }
         </style>
 
-        <div class="horizontal layout end-justified center">
+        <div class="container">
```

Comment:
Currently, in aspera ftests, we have to use the `icon` in order to select the correct `paper-icon-button` to navigate pages. It would be useful to take take this opportunity and add names or ids to each `paper-icon-button`, if you don't mind 🙂